### PR TITLE
move to nanoAODv7

### PR DESCRIPTION
**Pull request by nanlu06**
1. set DY rate parameter's range to 0.1 ~ 5
2. remove b-tag uncertainty "CMS_btag_Hfstats1","CMS_btag_Hfstats2" because of very small uncertainty with negligible effect and found to cause potential fit failing.
3. remove pile jet ID SF unc temporarily to sync with Pisa group datacard

### CODE REVIEW

**Comment by nanlu06** [comment]
/storage/user/nlu/hmm/nanoAODv7_2017/out/baseline/datacards/2017

Validation datacards: 0.92 sigma

File: tests/hmm/plotting.py [hunk 1]
```diff
@@ -1092,8 +1092,8 @@ def PrintDatacard(categories, dict_procs, era, event_counts, filenames, ofname):
         dcof.write("RZ rateParam {0} dy_2j 1 \n".format(cat.full_name)) 
         #dcof.write("REWZ rateParam {0} ewk_lljj_mll50_mjj120 1 \n".format(cat.full_name))
     elif ("h_peak" in cat.full_name) or ("h_sideband" in cat.full_name):
-        dcof.write("R_01j_{1} rateParam {0} dy_m105_160_01j 1 \n".format(cat.full_name,str(era)))           
-        dcof.write("R_2j_{1} rateParam {0} dy_m105_160_2j 1 \n".format(cat.full_name,str(era)))
+        dcof.write("R_01j_{1} rateParam {0} dy_m105_160_01j 1 [0.1,5] \n".format(cat.full_name,str(era)))           
+        dcof.write("R_2j_{1} rateParam {0} dy_m105_160_2j 1 [0.1,5] \n".format(cat.full_name,str(era)))
```
Comment (irenedutta23) [inline tests/hmm/plotting.py:1096]: this range is here rather wide, shouldn't it be constrained more around 1? For eg 0.5 to 1.5 ?

Comment (nanlu06) [reply tests/hmm/plotting.py:1096]: Hi Irene, good question. If in the toy mass scan study, I see cases where the rateParam for DY is fitted to even ~0.2 So I'm trying to keep it wide here. but yeah if you see issues with 0.1~5 later we can change it to a smaller range.

Comment (irenedutta23) [reply tests/hmm/plotting.py:1096]: ok sounds good

**Pull request merged**